crypto/src/vrf/schnorrkel/mod.rs

```diff
@@ -0,0 +1,158 @@
+use crate::random::{CryptoRng, Rng};
+use parity_scale_codec::{Decode, Encode};
+
+const EXPECTED_PUBKEY_LEN: usize = 32;
```

Comment:
I'd personally just call this `PUBKEY_LEN`

Reply:
Alrighty